src/63createvertex.js

```diff
@@ -250,123 +209,33 @@ yy.CreateGraph.prototype.execute = function (databaseid, params, cb) {
 		}
 	});
 
-	if (cb) {
-		res = cb(res);
-	}
-
-	return res;
+	return cb ? cb(res) : res;
 
-	// Find vertex by name
 	function findVertex(name) {
-		var objects = alasql.databases[alasql.useid].objects;
-		for (var k in objects) {
-			if (objects[k].name === name) {
-				return objects[k];
-			}
-		}
-		return undefined;
+		return Object.values(alasql.databases[alasql.useid].objects).find((obj) => obj.name === name);
```

Comment:
We simply cant be making copies of all the data in an object for operations like this. We need to keep the original or find a way where we can search the original object without making a copy.